addon/components/bs-alert.js

```diff
@@ -95,22 +84,12 @@ export default class Alert extends Component {
    * @default true
    * @public
    */
-  @defaultValue
+  @arg
   fade = true;
 
-  /**
-   * Computed property to set the alert class to the component div. Will be false when dismissed to have the component
-   * div (which cannot be removed form DOM by the component itself) without any markup.
```

Comment:
I think actually it made things easier, as the element that receives CSS transitions is always present. But I might be confused again. Anyway, I though that changing this could be seen as a breaking change in some edge cases, so I didn't touch it yet. I remember I tried, but got failing tests, so reverted...

Reply:
I see. Maybe something we can revisit after we have discussed the controlled vs. uncontrolled state.